tests/components/advantage_air/test_climate.py

```diff
@@ -73,18 +87,22 @@ async def test_climate_async_setup_entry(hass, aioclient_mock):
     assert len(aioclient_mock.mock_calls) == 5
     assert aioclient_mock.mock_calls[-2][0] == "GET"
     assert aioclient_mock.mock_calls[-2][1].path == "/setAircon"
+    data = loads(aioclient_mock.mock_calls[-2][1].query["json"])
+    assert data["ac1"]["info"]["state"] == ADVANTAGE_AIR_STATE_OFF
     assert aioclient_mock.mock_calls[-1][0] == "GET"
     assert aioclient_mock.mock_calls[-1][1].path == "/getSystemData"
 
     await hass.services.async_call(
         CLIMATE_DOMAIN,
         SERVICE_SET_FAN_MODE,
-        {ATTR_ENTITY_ID: [entity_id], ATTR_FAN_MODE: FAN_OFF},
+        {ATTR_ENTITY_ID: [entity_id], ATTR_FAN_MODE: FAN_LOW},
         blocking=True,
     )
     assert len(aioclient_mock.mock_calls) == 7
     assert aioclient_mock.mock_calls[-2][0] == "GET"
     assert aioclient_mock.mock_calls[-2][1].path == "/setAircon"
+    data = loads(aioclient_mock.mock_calls[-2][1].query["json"])
+    assert data["ac1"]["info"]["fan"] == HASS_FAN_MODES[FAN_LOW]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)